src/DonorProfiles/Helpers/LocationList.php

```diff
@@ -11,12 +13,12 @@ class LocationList {
 	public static function getCountries() {
 		$countries = give_get_country_list();
 		unset( $countries[''] );
-		return FormatList::fromKeyValue( $countries );
+		return FormatObjectList\Factory::fromKeyValue( $countries );
```

Comment:
@henryholtgeerts quick note: I updated the implementation a little, so you'll need to call `->format()` to get the formatted list. The `Factory` returns an instance of the formatter class.

Reply:
Ah - I see! Tried that implementation first but IDE was yelling at me because the doc blocks for FormatObjectList/Factory indicate that it would return an array (not an object). Could you update doc blocks your package real quick to reflect this and I'll make the adjustment here as well?

Reply:
I've corrected the implementation here, but still suggest updating the doc blocks in your package to correctly reflect the values returned by Factory methods. Ref: https://github.com/impress-org/givewp/pull/5486/commits/5e7f868d222d1c107e1bf0d37d3bd0da7278e51c